docs/orchestration/flow_config/run_configs.md

```diff
@@ -58,6 +58,22 @@ register a flow with no labels it will only be picked up by Agents which also
 do not have labels specified.
 :::
 
+:::tip Automatic Labels
+If your Flow is configured for [Local Storage](storage.md#local), which is the
+default, then flow runs generated from it will automatically be given a label
+matching the hostname of the machine where the flow run was created. This behavior can be disabled
+by adding the following to your `config.toml`:
```

Comment:
An additional comment on this - this is valid, but we should also call out the less blunt approach:
```python
flow.storage = Local(add_default_labels=False)
```
which can be configured on a flow-by-flow basis.